src/main/java/com/epam/ta/reportportal/binary/impl/AttachmentBinaryDataServiceImpl.java

```diff
@@ -81,7 +81,7 @@ public AttachmentBinaryDataServiceImpl(ContentTypeResolver contentTypeResolver,
 	@Override
 	public Optional<BinaryDataMetaInfo> saveAttachment(AttachmentMetaInfo metaInfo, MultipartFile file) {
 		Optional<BinaryDataMetaInfo> result = Optional.empty();
-		try {
+		try (InputStream inputStream = file.getInputStream()) {
```

Comment:
Also required in:
https://github.com/reportportal/commons-dao/pull/614/files#diff-949ed9fb69ce1aa64ddcf74421ffdd7aL169

And, probably in other DataStoreServices

Reply:
done